recipes-devtools/python/python3-lxa-iobus_git.bb

```diff
@@ -0,0 +1,36 @@
+DESCRIPTION = "Daemon which interfaces IOBus-devices from Linux Automation GmbH with test-automation tools like labgrid."
+HOMEPAGE = "https://github.com/linux-automation/lxa-iobus"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=0674f4b6076ccd96a8b400a746f71dd3"
+
+RDEPENDS_${PN} = " \
+    bash \
+    git \
+    python3-aiohttp \
+    python3-aiohttp-json-rpc \
+    python3-can \
+    python3-janus \
+"
+
+SRC_URI = " \
+    git://github.com/linux-automation/lxa-iobus.git;branch=master \
+    file://80_can0-iobus.link \
+    file://80_can0-iobus.network \
+    file://lxa-iobus.service \
+    "
+
+# Version 0.3
+SRCREV = "0261687d428171ed134399b66506ce7958bcddca"
+S = "${WORKDIR}/git"
+
+DEPENDS += "python3-setuptools-scm-native"
+
+inherit setuptools3
+
+do_install_append() {
+    install -D -m0644 ${WORKDIR}/80_can0-iobus.link ${D}${libdir}/systemd/network/80_can0-iobus.link
+    install -D -m0644 ${WORKDIR}/80_can0-iobus.network ${D}${libdir}/systemd/network/80_can0-iobus.network
```

Comment:
The link/network files are board specific configuration and should not be contained within this recipe.

Reply:
Where do they come from?

Reply:
This is from the labgrid LXATAC integration.

Reply:
I mean, I cannot find these files in the PR. Or will these lines be removed in a further PR update step?